api/signup.go

```diff
@@ -2,19 +2,17 @@ package api
 
 import "github.com/infrahq/infra/internal/validate"
 
-type SignupEnabledResponse struct {
-	Enabled bool `json:"enabled"`
-}
-
 type SignupRequest struct {
 	Name     string `json:"name"`
 	Password string `json:"password"`
+	Org      string `json:"org"`
 }
 
 func (r SignupRequest) ValidationRules() []validate.ValidationRule {
 	return []validate.ValidationRule{
 		validate.Required("name", r.Name),
 		validate.Required("password", r.Password),
+		validate.Required("org", r.Org),
```

Comment:
I'll separate this out into orgName and orgDomain, it's just not in this branch yet from #2779.

Reply:
Yeah, originally I was thinking just Org Name, but this makes more sense.